spec/services/throttle_spec.rb

```diff
@@ -69,12 +69,22 @@
 
   describe '#increment!' do
     subject(:throttle) { Throttle.new(target: 'aaa', throttle_type: :idv_doc_auth) }
+    let(:max_attempts) { 1 }
 
     it 'increments attempts' do
       expect(throttle.attempts).to eq 0
       throttle.increment!
       expect(throttle.attempts).to eq 1
     end
+
+    it 'does nothing if already throttled' do
+      expect(throttle.attempts).to eq 0
+      throttle.increment!
+      expect(throttle.attempts).to eq 1
+      expect(throttle.throttled?).to eq(true)
+      throttle.increment!
+      expect(throttle.attempts).to eq 1
```

Comment:
Would it be worth adding a check here that the expiration also does not change? Maybe using [time traveling spec helpers](https://api.rubyonrails.org/v5.2.4/classes/ActiveSupport/Testing/TimeHelpers.html) to try incrementing the throttle after a simulated delay.

Reply:
Good idea! Added here: a1df94f14e962167f31bb421ba2ca269514b382b